charts/qdrant/Chart.yaml

```diff
@@ -13,30 +13,11 @@ maintainers:
     url: https://github.com/qdrant
 icon: https://qdrant.github.io/qdrant-helm/logo_with_text.svg
 type: application
-version: 0.9.0
-appVersion: v1.9.1
+version: 0.9.4
```

Comment:
The version is entirely separate from the app version, so it should just be bumped according to normal semver rules, to 0.9.1

Reply:
Pushed version fix